01 - Getting Started.ipynb

```diff
@@ -24,13 +24,13 @@
     "### `conda` / `mamba`\n",
```

Comment:
<p>When I run this, it created a temp file <code>.ddb.wal</code>, I would add that to the <code>.gitignore</code> and if worth it mention that is a temp file and not to worry about it for the purpose of this tutorial.    </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>